Sources/PIALibrary/Client+Preferences.swift

```diff
@@ -307,6 +307,10 @@ extension Client {
             allMaps[vpnType] = customConfiguration.serialized()
             accessedDatabase.plain.vpnCustomConfigurationMaps = allMaps
         }
+
+        public func setVpnTypeToWireguard() {
+            self.vpnType = PIAWGTunnelProfile.vpnType
+        }
```

Comment:
Similar to the comment above: we need to make sure that this is not performed on tvOS, only on iOS
Should we set the availability of this method to only iOS? So then we make sure that we can't use this on tvOS, what do you think?

Reply:
Sure, I can wrap it around the iOS preprocessor. Just out of curiosity are we sharing mobile-ios logic between ios and tvos?

Reply:
Yes, we are using the same connection logic on tvOS. We will migrate to use the VPNManager when the project is done